apis/kueue/v1beta1/admissioncheck_types.go

```diff
@@ -97,9 +97,6 @@ const (
 	// AdmissionCheckActive indicates that the controller of the admission check is
 	// ready to evaluate the checks states
 	AdmissionCheckActive string = "Active"
-
-	// FlavorIndependentAdmissionCheck indicates if the AdmissionCheck cannot be applied at ResourceFlavor level.
-	FlavorIndependentAdmissionCheck string = "FlavorIndependent"
```

Comment:
It does not seem related

Reply:
Yeah, I think it was missed and not removed in https://github.com/kubernetes-sigs/kueue/pull/4995.

Reply:
Created PR https://github.com/kubernetes-sigs/kueue/pull/5036.